### DISCUSSION TITLE
download all data in one csv [Oct 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4037] [Discord Threads]
> 🙋‍♂️ **prasanth**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### how many records you have?  
#  
> 🙋‍♂️ **prasanth**  
#### <@495466997304262658>   
#### i have a table with entries more than 100k and it will increase further more   
####    
#### on applying a filter it gave me a count of 20k    
####    
#### on downloading it got downloaded into around 5 file with each havin close to 4k rows  
#  
> 🙋‍♂️ **wkw**  
#### i think it's not possible. however, you can increase `NC_EXPORT_MAX_TIMEOUT` to fetch as much data as possible within that timeframe. by default it is 5000 (in ms).    
####    
#### See https://docs-dev.nocodb.com/getting-started/installation#environment-variables  
#  
> 🙋‍♂️ **prasanth**  
#### yeah i have checked that env var. is there any max timeout value? it is not mentioned in docs.  
#  
> 🙋‍♂️ **wkw**  
#### we are not limiting  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1024343013008347237)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
